### ISSUE TITLE
connectors and acceptors exposure supports Route but not Ingress

### ISSUE THREAD

**Issue by gaohoward** [artemiscloud/activemq-artemis-operator#131]
with ingress/routes, TLS is required as the endpoint is http only, with pass through tls and SNI, it can work for any tcp layered protocol, ie: expose=true and sslEnabled=true

These routes/ingresses are not currently reconciled, such that flipping sslEnabled is not respected

**Comment by gtully** [comment]
btw: I am not seeing the use case for exposing connectors

**Comment by gaohoward** [comment]
I don't see it either.

**Comment by gtully** [comment]
opened https://github.com/artemiscloud/activemq-artemis-operator/issues/401 to track deprecation

**Comment by brusdev** [comment]
The `--enable-ssl-passthrough` flag enables the SSL Passthrough feature, which is disabled by default. This is required to enable passthrough backends in Ingress objects. For further details see https://kubernetes.github.io/ingress-nginx/user-guide/tls/#ssl-passthrough

```
minikube addons enable ingress && minikube kubectl -- patch deployment -n ingress-nginx ingress-nginx-controller --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value":"--enable-ssl-passthrough"}]'
```

**Comment by brusdev** [comment]
Fixed in v1.0.11
